Makefile

```diff
@@ -93,11 +92,6 @@ codestyle-fix: $(NODE_MODULES) ## Run and fix style check errors
 	@$(JSCS) $(ALL_FILES) --fix
 
 
-.PHONY: nsp
```

Comment:
Are we removing this `nsp` target because now this functionality is included in the new `npm audit` command?

Reply:
We used to have a script that would take the results of nsp and create a badge (safe/unsafe) in the README - but it was pretty misleading since transitive dependencies can change all the time. We removed it in various other repos but probably never got around to it here. 

I think `npm audit` is likely the long term story.